packages/airnode-examples/integrations/authenticated-coinmarketcap/config.example.json

```diff
@@ -11,7 +11,12 @@
           "url": "${PROVIDER_URL}"
         }
       },
-      "type": "evm"
+      "type": "evm",
+      "chainOptions": {
+        "txType": "2",
+        "baseFeeMultiplier": "2",
+        "priorityFeeGwei": "3.12"
```

Comment:
`priorityFeeInGwei` imo. Also, what if we had a `priorityFee` instead that expects `"3.12 gwei"` or `3120000000 wei`? I've never seen this, but having it in Gwei by default feels too opinionated but having a lot of optional parameters in the form of `priorityFeeInGwei`, `priorityFeeInWei`, etc. is not nice.

Reply:
`ethers` has a facility to parse amounts where the unit is specified, so it is feasible to specify the Priority Fee as "3.12 gwei", leaving the unit up to the user. I can prepare an amendment that uses this approach.

Reply:
Maybe this is the proper way of doing it though
```
"priorityFee": { "value": "3.12", "unit": "gwei" }
```

Reply:
Good idea, I'll do this.